.github/ISSUE_TEMPLATE/Bug_report.md

```diff
@@ -10,6 +10,8 @@ about: Report something working incorrectly
 
 #### Provide the steps to reproduce
 1. Run LH on <affected url>
+  
+<!-- If your page is only local, or is liable to change, consider uploading a repro to https://surge.sh/ so that we can more easily debug the problem. -->
```

Comment:
```suggestion
<!-- If your page is only local, or is liable to change, consider uploading a repro to https://surge.sh/ or creating one at https://jsbin.com/ so that we can more easily debug the problem. -->
```